f421makefile.mk

```diff
@@ -1,7 +1,7 @@
 
 TARGETS_F421 :=  WRAITH32_F421 AIKON_SINGLE_F421 AIKON_55A_F421 FOXEER_F421 HAKRC_K_F421 \
 	             HAKRC_G_F421 TEKKO32_F421 AT32DEV_F421 DARWIN_F421 NEUTRON_F421 AT32SLOTCAR_F421\
-				 HAKRC_G23_F421 NEUTRON_MINIAIO_F421 FOXEER_F421\  
+				 HAKRC_G23_F421 NEUTRON_MINIAIO_F421 BAYCKRC_S_55A_F421 FOXEER_F421\  
```

Comment:
this isn't needed with the new makefile system, can you please rebase the PR on main?

Reply:
Okay, everything will be adjusted according to your latest coding structure. If you need me to do anything, just give me instructions. Thank you.

Reply:
> this isn't needed with the new makefile system, can you please rebase the PR on main?

The modifications in the makefile have been removed.